pom.xml

```diff
@@ -71,6 +71,22 @@
     </dependencies>
   </dependencyManagement>
   <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
```

Comment:
Please remove this, you will be fighting the upper dependency requirement (since #90 was merged) and the HTTP components plugin already takes care of it for you: https://github.com/jenkinsci/apache-httpcomponents-client-4-api-plugin/blob/master/pom.xml

Reply:
+1

Reply:
done